### ISSUE TITLE
[JOSS] statement of need - dask

### ISSUE THREAD

**Issue by platipodium** [pmlmodelling/nctoolkit#57]
For your statement of need 

> For example, the Coupled Model Intercomparison Project Phase 6 (O’Neill et al., 2016), produced approximately 20 PB of publicly available data (Petrie et al., 2021).

Please clarify why you would not rely on technologies like `dask` that would distribute the work, especially with those large data files.

**Comment by platipodium** [comment]
https://github.com/openjournals/joss-reviews/issues/5494

**Comment by robertjwilson** [comment]
The simple answer is that the package is currently developed based on project needs. Using `dask` is something on our minds, but it will need to align with ongoing project needs.

**Comment by platipodium** [comment]
Could you then make a performance statement, or a statement on the limitations of your software and its multiprocessing capabilities?

**Comment by robertjwilson** [comment]
I've added a statement on parallel processing to the paper. https://github.com/pmlmodelling/nctoolkit/commit/04e09b63d86f6f6f62c8166a1c9825876d5b084a

It's difficult to come up with a defensible performance statement for JOSS without spending days creating rigorous benchmarks, so I'd prefer to avoid it. And it's probably impossible to do given the tight word limit.

As raised here, I've got some benchmarks that suggest similar performance to xarray: https://github.com/pmlmodelling/nctoolkit/issues/81

Though I've yet to test performance on multi-file datasets in xarray versus nctoolkit.
